ansible/roles/atmo-mount-volume/tasks/main.yml

```diff
@@ -16,9 +15,17 @@
     fstype: '{{ VOLUME_DEVICE_TYPE }}'
     src: '{{ VOLUME_DEVICE }}'
     state: 'mounted'
-    name: '{{ VOLUME_MOUNT_LOCATION }}'
+    path: '{{ VOLUME_MOUNT_LOCATION }}'
   register: mount_result
 
 - name: Debug mount result
   debug:
     var: mount_result
+
+- name: Change directory ownership
+  file:
+    state: directory
+    path: "{{ VOLUME_MOUNT_LOCATION }}"
+    recurse: yes
+    owner: "{{ ATMOUSERNAME }}"
+    group: "iplant-everyone"
```

Comment:
Good point. Is there a way to do this without having to add a new variable to all the secret repos?